### PR TITLE
page: mcq page added with functionality

### PR DESCRIPTION
**Pull request by manojsarna**
- [x] MCQ Page Added
- [x] Loaders Added
- [x] Functionality Added - Score , Quit Quiz , Next Question

image
image

### CODE REVIEW

**Review by pankaj-prog — APPROVED** [review]
Didn't find any issue. 
Suggestion
- use absolute imports

**Review by iRohitGaur — APPROVED** [review]
Good to merge 👍🏻

File: src/pages/quiz/Questions.jsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                        setScore((score) => score + 2);
+                      }
+                    }
+                  }}
+                >
+                  {option}
+                </button>
+              ))}
+            </div>
+            <div></div>
```
Comment (gauravsinhaweb) [inline src/pages/quiz/Questions.jsx:97]: can remove this `div` if not in use.

File: src/pages/quiz/Rules.jsx [hunk 2]
```diff
@@ -1,35 +1,41 @@
...
+          <h1 className="rules-heading">Rules</h1>
+          <ol className="sm-main-rules-container">
+            <li>This quiz consists of 5 multiple-choice questions.</li>
+            <li>Points will be awarded : 2 points for each right answer.</li>
+            <li>Each multiple choice question has only one correct answer.</li>
+            <li>
+              To start, click the Start Quiz button. When finished, click the
+              Submit button.
+            </li>
+          </ol>
```
Comment (gauravsinhaweb) [inline src/pages/quiz/Rules.jsx:20]: rules can be different for different categories, so better to get rules data from the backend.

File: src/pages/quiz/questions.css [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  justify-content: space-between;
+}
+
+.btn-end-quiz {
+  text-decoration: none;
+  border-radius: 3rem;
+  background: var(--COLOR-RED);
+  color: var(--COLOR-WHITE-ALWAYS);
+  border: 1px solid var(--COLOR-RED);
+  cursor: pointer;
```
Comment (gauravsinhaweb) [inline src/pages/quiz/questions.css:102]: good  use of var colors

**Review by gauravsinhaweb — APPROVED** [review]
code looks clean. some suggestions I have mentioned can be improved.

**Pull request merged**